.editorconfig

```diff
@@ -0,0 +1,22 @@
+﻿[*.cs]
```

Comment:
I still don't think we should have this file as it could be affecting the rules we have in place for other projects set by StyleCop. I checked locally and if we remove this file, you'd need to do three things:

- Add the [`<DisableDocRequirement>`](https://github.com/microsoft/Oryx/blob/main/src/BuildScriptGenerator/BuildScriptGenerator.csproj#L17) tag to your `Automation.csproj` file to dismiss the errors about the XML comments
- Add inline attribute suppressions to the file complaining about the class types not matching the file name (the file where you have all of the nested deserialization classes)
- Clean up the additional errors that remain (curly braces on a new line, empty lines needing to be removed, etc.)